components/openzwave-adapter/src/lib.rs

```diff
@@ -107,18 +107,18 @@ trait RangeChecker {
     fn should_send(&self, &Value, EventType) -> bool;
 }
 
-impl RangeChecker for Option<Box<Range>> {
+impl RangeChecker for Option<Value> {
     fn should_send(&self, value: &Value, event_type: EventType) -> bool {
         match *self {
             None => event_type == EventType::Enter, // no range means we send only Enter events
-            Some(ref range) => range.contains(value)
+            Some(ref range) => range == value // FIXME: This won't scale up to interesting ranges.
         }
     }
 }
 
-impl RangeChecker for Range {
+impl RangeChecker for Value {
     fn should_send(&self, value: &Value, _: EventType) -> bool {
-        self.contains(value)
+        self == value // FIXME: This won't scale up to interesting ranges.
```

Comment:
Well, this `FIXME` is actually a reminder that `should_send` only works for simple cases. For more complicated cases, well, a different algorithm will be needed. As I have no clue about which cases we are going to handle in the future, I don't really see how I can be much more specific.